lib/gems/pending/appliance_console/database_configuration.rb

```diff
@@ -140,6 +141,7 @@ def friendly_inspect
 Username: #{username}
 Database: #{database}
 FRIENDLY
+      output << "Port:     #{port}\n" if port
```

Comment:
I think we should leave it. Display the default if being used as the default value.